### PR TITLE
large pep8 cleanup in interfaces

### PR DESCRIPTION
**Pull request by fchapoton**
just fixing many small code details in `interfaces` folder

### :memo: Checklist

- [x] The title is concise, informative, and self-explanatory.
- [x] The description explains in detail what this PR is about.

### CODE REVIEW

File: src/sage/interfaces/magma.py [hunk 1]
```diff
@@ -1303,33 +1303,37 @@ def _object_class(self):
     # It's very painful using the interface without this.
     def _left_list_delim(self):
         """
-        Return the left sequence delimiter in Magma. Despite the name in
+        Return the left sequence delimiter in Magma.
+
+        Despite the name in
         this function, this is really the least painful choice.
```
Comment (kwankyu) [inline src/sage/interfaces/magma.py:1310]: reformat 1308-1309?

File: src/sage/interfaces/magma.py [hunk 2]
```diff
@@ -1303,33 +1303,37 @@ def _object_class(self):
...
 
         EXAMPLES::
 
             sage: magma._left_list_delim()
             '['
         """
-        #return "[*"
+        # return "[*"
         return "["
```
Comment (kwankyu) [inline src/sage/interfaces/magma.py:1316]: I think these commented code with no explanation why they are there should be eliminated.

File: src/sage/interfaces/magma.py [hunk 3]
```diff
@@ -1303,33 +1303,37 @@ def _object_class(self):
...
 
     def _right_list_delim(self):
         """
-        Return the right sequence delimiter in Magma. Despite the name in
+        Return the right sequence delimiter in Magma.
+
+        Despite the name in
         this function, this is really the least painful choice.
```
Comment (kwankyu) [inline src/sage/interfaces/magma.py:1324]: reformat?

File: src/sage/interfaces/matlab.py [hunk 4]
```diff
@@ -149,7 +149,7 @@
 #
 #  The full text of the GPL is available at:
 #
-#                  http://www.gnu.org/licenses/
+#                  https://www.gnu.org/licenses/
 ##############################################################################
```
Comment (kwankyu) [inline src/sage/interfaces/matlab.py:154]: `# *****`?

File: src/sage/interfaces/mupad.py [hunk 5]
```diff
@@ -90,7 +90,7 @@
 #
 #  Distributed under the terms of the GNU General Public License (GPL)
 #
-#                  http://www.gnu.org/licenses/
+#                  https://www.gnu.org/licenses/
 #############################################################################
```
Comment (kwankyu) [inline src/sage/interfaces/mupad.py:95]: `# ****`?

File: src/sage/interfaces/polymake.py [hunk 6]
```diff
@@ -277,17 +277,16 @@ def _coerce_impl(self, x, use_special=True):
...
             for k, v in x.items():
                 k = convert(k)
                 v = convert(v)
                 z[k] = v
                 A.append("{}=>{}".format(k.name(), v.name()))
             r = self.new("{" + ",".join(A) + "}")
-            r.__sage_dict = z # do this to avoid having the entries of the list be garbage collected
+            r.__sage_dict = z
+            # this to avoid having the entries of the list be garbage collected
             return r
```
Comment (kwankyu) [inline src/sage/interfaces/polymake.py:291]: A comment should go before the code.

File: src/sage/interfaces/polymake.py [hunk 7]
```diff
@@ -851,7 +850,7 @@ def new_object(self, name, *args, **kwds):
 
 
 ########################################
-## Elements
+#  Elements
```
Comment (kwankyu) [inline src/sage/interfaces/polymake.py:856]: If you are willing, I would suggest
```
# --------
# Elements
# --------
```
as the standard style for header comments

File: src/sage/interfaces/r.py [hunk 8]
```diff
@@ -266,9 +266,10 @@
 #
 #  Distributed under the terms of the GNU General Public License (GPL)
 #
-#                  http://www.gnu.org/licenses/
+#                  https://www.gnu.org/licenses/
 #
 ##########################################################################
+import os
```
Comment (kwankyu) [inline src/sage/interfaces/r.py:272]: `# ****`?

File: src/sage/interfaces/r.py [hunk 9]
```diff
@@ -2047,7 +2053,6 @@ def reduce_load_R():
     return r
 
 
-import os
 def r_console():
```
Comment (kwankyu) [inline src/sage/interfaces/r.py:2053]: Is this safe?

File: src/sage/interfaces/scilab.py [hunk 10]
```diff
@@ -192,7 +192,7 @@
 #
 #  The full text of the GPL is available at:
 #
-#                  http://www.gnu.org/licenses/
+#                  https://www.gnu.org/licenses/
 ##############################################################################
```
Comment (kwankyu) [inline src/sage/interfaces/scilab.py:197]: `# ***`?

File: src/sage/interfaces/tests.py [hunk 11]
```diff
@@ -41,38 +41,41 @@
...
     try:
         t = cputime()
         w = walltime()
         v = []
         for i in range(num):
-            if i % step == 0:
-                sys.stdout.write('%s ' % i)
+            if not i % step:
+                sys.stdout.write('{i} ')
                 sys.stdout.flush()
```
Comment (kwankyu) [inline src/sage/interfaces/tests.py:63]:
```
f'{i} '
```
?

File: src/sage/interfaces/tides.py [hunk 12]
```diff
@@ -18,7 +18,7 @@
     #
     #  Distributed under the terms of the GNU General Public License (GPL):
     #
-    #                  http://www.gnu.org/licenses/
+    #                  https://www.gnu.org/licenses/
     ##########################################################################
```
Comment (kwankyu) [inline src/sage/interfaces/tides.py:23]: `# ***`?

**Comment by kwankyu** [comment]
Otherwise lgtm.

**Comment by kwankyu** [comment]
Please rebase as well.

File: src/sage/interfaces/r.py (hunk 9 above)
Comment (fchapoton) [reply src/sage/interfaces/r.py:2053]: it was a duplicate import

**Pull request merged**